src/three-components/FootprinterModel.tsx

```diff
@@ -3,8 +3,17 @@ import { createJSCADRenderer } from "jscad-fiber"
 import { jscadPlanner } from "jscad-planner"
 import { useMemo } from "react"
 import { JscadModel } from "./JscadModel"
+import * as modeling from "@jscad/modeling"
 
-const { createJSCADRoot } = createJSCADRenderer(jscadPlanner as any)
+const jscadModule = {
+  ...jscadPlanner,
+  hulls: {
+    hull: modeling.hulls.hull,
+    hullChain: modeling.hulls.hullChain,
+  },
+}
+
+const { createJSCADRoot } = createJSCADRenderer(jscadModule as any)
```

Comment:
this is the right idea, but you should actually PR to `jscad-planner`, if we merged this it would bug out in weird ways. `jscad-planner` is like `jscad` but instead of doing the 3d operations directly it stores them